internal/validator/fabric_v14_validator.go

```diff
@@ -0,0 +1,35 @@
+package validator
+
+import (
+	"github.com/meshplus/bitxhub/internal/ledger"
+	"github.com/meshplus/bitxhub/pkg/vm/wasm/wasmlib"
+	"github.com/sirupsen/logrus"
+)
+
+// Validator is the instance that can use wasm to verify transaction validity
+type FabV14Validator struct {
+	ledger ledger.Ledger
+	logger logrus.FieldLogger
+}
+
+// New a validator instance
+func NewFabV14Validator(ledger ledger.Ledger, logger logrus.FieldLogger) *FabV14Validator {
+	return &FabV14Validator{
+		ledger: ledger,
+		logger: logger,
+	}
+}
+
+// Verify will check whether the transaction info is valid
+func (vlt *FabV14Validator) Verify(address, from string, proof []byte, validators string) (bool, error) {
+	vInfo, err := wasmlib.UnmarshalValidatorInfo([]byte(validators))
```

Comment:
Why are `validators'Unmarshal` and `ValidateV14 ` methods inside the `wasmlib` package?

Reply:
The validation method will be reorganized later. The validation module will be extracted as an independent part and where the lib functions placed will be determined then.